Dallia_Sintique/sistema_notas.py

```diff
@@ -0,0 +1,177 @@
+from dataset_alunas import dataset
+
+def obter_opcao():
+    codigo_opcao = 0
+
+    while codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+        try:
+            codigo_opcao = int(input("\nEscolha uma opção:\n"
+                                    "1 - Incluir uma nova aluna\n"
+                                    "2 - Consultar lista de alunas\n"
+                                    "3 - Consultar faltas da aluna\n"
+                                    "4 - Consultar notas da aluna\n"
+                                    "5 - Consultar status de aprovação\n"
+                                    "6 - Sair do sistema\n"
+                                    "Opção: "))
+
+            if codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+                print("Opção inválida. Por favor, escolha uma opção válida (1 a 5).\n")
+        except ValueError:
+            print("Entrada inválida. Por favor, digite um número inteiro.\n")
+
+        return codigo_opcao
+
+def obtem_nome(pergunta):
+  while True:
+    entrada_usuário = str(input(pergunta)).lstrip().rstrip()
```

Comment:
1. Quando utilizamos o input, já estamos recebendo uma string, assim, não precisamos utilizar uma conversão de dados str :)
2. Para não precisar utilizar um lstrip e um rstrip, podemos usar o strip, e ele já remove os espaços no início e final da string

**Sugestão:**
```suggestion
    entrada_usuário = input(pergunta).strip()
```